Bugzilla/DB/Mysql.pm

```diff
@@ -330,15 +330,6 @@ sub bz_setup_database {
   }
 
   if ($self->utf8_charset eq 'utf8mb4') {
-    my %global = map {@$_}
-      @{$self->selectall_arrayref(q(SHOW GLOBAL VARIABLES LIKE 'innodb_%'))};
-    my $utf8mb4_supported
-      = $global{innodb_file_format} eq 'Barracuda'
-      && $global{innodb_file_per_table} eq 'ON'
-      && $global{innodb_large_prefix} eq 'ON';
-
-    die install_string('mysql_innodb_settings') unless $utf8mb4_supported;
```

Comment:
In the bug/commit message/PR description, can you elaborate why this is no longer needed? Perhaps link to the specific part of the release notes that obsolete this?